nucypher/characters/lawful.py

```diff
@@ -942,19 +932,19 @@ def _get_cleartexts(self,
     def _handle_treasure_map(self,
                              publisher_verifying_key: PublicKey,
                              label: bytes,
-                             treasure_map: Optional['TreasureMap'] = None,
+                             enc_treasure_map: Optional['EncryptedTreasureMap'] = None,
                              ) -> 'TreasureMap':
-        if treasure_map is not None:
-            self._try_orient(treasure_map, publisher_verifying_key)
-            # self.treasure_maps[treasure_map.public_id()] = treasure_map # TODO: Can we?
+        if enc_treasure_map is not None:
```

Comment:
Unless this can possibly be assigned to a CS constant, this can be a simple falsy check.